resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/submitsteps/PythonStep.scala

```diff
@@ -27,15 +27,19 @@ private[spark] class PythonStep(
     filesDownloadPath: String) extends KubernetesSubmissionStep {
 
   override def prepareSubmission(driverSpec: KubernetesDriverSpec): KubernetesDriverSpec = {
+    val resolvedOtherPyFilesString = if (otherPyFiles.isEmpty) {
```

Comment:
Not exactly, it would need to be `null` unless you have docker parse, but docker doesnt do if: else: blocks so thats why this woudn't be helpful per say